client/src/theme/styles/components/backend/project-collection/collectionList.js

```diff
@@ -26,6 +26,11 @@ export default `
         breakpointLessOne(75),
         "max"
       )}
+
+      ul {
+        max-height: 1270px;
+        overflow-y: scroll;
```

Comment:
Always use `auto` over `scroll` unless you are certain that you want a scrollbar to be visible under all circumstances. `auto` will only show one if there is actually content overflowing the container.